src/Tizen.NUI.Components/Style/Extension/LottieButtonStyle.cs

```diff
@@ -60,15 +60,11 @@ public override void CopyFrom(BindableObject bindableObject)
         {
             base.CopyFrom(bindableObject);
 
-            LottieButtonStyle style = bindableObject as LottieButtonStyle;
-
-            if (style == null)
+            if (bindableObject is LottieButtonStyle lottieButtonStyle)
             {
-                return;
+                LottieUrl = lottieButtonStyle.LottieUrl;
+                PlayRange = lottieButtonStyle.PlayRange?.Clone();
```

Comment:
PlayRange is LottieFrameInfo type.
LottieFrameInfo  has Clone method in the latest master branch?

Reply:
`PlayRange` is `Selector` type. (`public Selector<LottieFrameInfo> PlayRange { get; set; }`)
We don't need `ISelectorItem` patch.